validator/client/service.go

```diff
@@ -55,7 +55,7 @@ func NewValidatorService(ctx context.Context, cfg *Config) (*ValidatorService, e
 func (v *ValidatorService) Start() {
 	pubkeys := make([][]byte, 0)
 	for i := range v.keys {
-		log.WithField("publicKey", fmt.Sprintf("%#x", v.keys[i].PublicKey.Marshal())).Info("Initializing new validator service")
+		log.WithField("pubKey", hex.EncodeToString(v.keys[i].PublicKey.Marshal())[:12]).Info("Initializing new validator service")
```

Comment:
the first time we start up the client, I think we should log out the whole public key. After that it is fine to truncate it

Reply:
+1